action/action-simulator/src/main/java/com/powsybl/action/simulator/tools/ActionSimulatorToolConstants.java

```diff
@@ -12,6 +12,7 @@
     public static final String CASE_FILE = "case-file";
     public static final String CONTINGENCIES = "contingencies";
     public static final String DSL_FILE = "dsl-file";
+    public static final String PARAMETERS_FILE = "parameters-file";
```

Comment:
I don't understand the purpose of this  PR as the  constant is  not used anywhere?